pkg/roachprod/install/cluster_synced.go

```diff
@@ -2597,24 +2597,33 @@ func (c *SyncedCluster) Get(
 		close(results)
 	}()
 
+	var errOnce sync.Once
+	var finalErr error
+	setErr := func(e error) {
+		if e != nil {
+			errOnce.Do(func() {
```

Comment:
Just for my own understanding, is there a particular reason we capture only the first error, does it matter if this becomes the last error?

Reply:
Yeah admittedly I'm not too sure either, I was just copying the behavior found in `Put()`. The commit for that doesn't give much insight either so I just played it safe and kept it.

>does it matter if this becomes the last error?

I'm guessing no and it just chooses one of the errors so as to not spam the user with an error per node. And maybe choosing the first one is more intentional than letting it repeatedly overwrite an err object 🤷

Reply:
Ah makes sense, I see the logic came from somewhere else. Thanks for updating!